README.md

```diff
@@ -41,44 +67,47 @@ Include a diagram describing the application
 - This model uses [MODEL 1] and [MODEL 2] which may not be available in all Azure regions. Check for [up-to-date region availability](https://learn.microsoft.com/azure/ai-services/openai/concepts/models#standard-deployment-model-availability) and select a region during deployment accordingly
     - We recommend using [SUGGESTED REGION]
 - ...
- 
-### Installation
- 
+
+#### Installation
+
 (ideally very short)
  
 - npm install [package name]
 - mvn install
```

Comment:
I think these steps would need to be after azd init, if they're trying to install the required packages for a project, since you need the package.json / requirements.txt?